zcash_proofs/Cargo.toml

```diff
@@ -9,8 +9,13 @@ authors = [
 bellman = { path = "../bellman" }
 blake2b_simd = "0.5"
 byteorder = "1"
+directories = { version = "1", optional = true }
```

Comment:
I added version 1 as a dependency instead of version 2, because version 1 has fewer dependencies.

Reply:
Is it still maintained?

Reply:
Version 2 is actively maintained, but has a dependency on `redox_users`, which pulls in a bunch of extraneous crates we don't actually need. It's unclear to me whether version 1 is maintained.